config/application.rb

```diff
@@ -16,6 +17,9 @@ class Application < Rails::Application
     # Load files in lib
     config.eager_load_paths << Rails.root.join('lib')
 
+    # set correct formatter for better error handling
```

Comment:
What is a `correct` formatter? What is `better error handling`? Explain why this formatter is needed.